frontend/src/pages/ActivityReport/Pages/components/Objective.js

```diff
@@ -132,7 +132,7 @@ const Objective = ({
                 <Dropdown
                   name="status"
                   onChange={onChange}
-                  value={status}
+                  value={status || 'Not Started'}
```

Comment:
The status here is sometimes null? Did you track down why it can be null sometimes? I would expect it to never be null since it has a value when created in the [Goal Picker Component](https://github.com/adhocteam/Head-Start-TTADP/blob/main/frontend/src/pages/ActivityReport/Pages/components/GoalPicker.js#L22).